cla_backend/apps/call_centre/admin/__init__.py

```diff
@@ -7,14 +7,34 @@
 from ..models import Operator, Caseworker, Organisation
 
 
+class OrganisationListFilter(admin.SimpleListFilter):
+    title = "Organisation"
+    parameter_name = "organisation"
+
+    def lookups(self, request, model_admin):
+        organisations = []
+        for organisation in Organisation.objects.all():
+            organisations.append((organisation.id, organisation.name))
+
+        if organisations:
+            organisations.append(("", "Unassigned"))
+        return organisations
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        kwargs = {}
+        if value:
+            kwargs["organisation__id"] = value
+        elif value == "":
+            kwargs["organisation__isnull"] = True
+
```

Comment:
It would be good to have a comment explaining that the other case is value being `None` and why and how that is implicitly handled